frontend/src/components/PlanHome/PlanHome.module.css

```diff
@@ -0,0 +1,333 @@
+  border-radius: 20px;
+}
+
+.inviteMemberContainer {
+  position: relative;
+
+  width: 250px;
+  height: 50px;
+  margin-left: 20px;
+
+  background-color: var(--green);
+  border-radius: 10px;
+  border-style: none;
+  cursor: pointer;
+
+}
+
+.inviteMemberContainer p {
+  position: absolute;
+  margin: 0;
+  top: 13px;
+  left: 70px;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 27px;
+  color: var(--white);
+}
+
+.inviteMemberContainer img {
+  position: absolute;
+  left: 20px;
+  top: 10px;
+
+  width: 30px;
+  height: 30px;
+  margin: 0;
+
+  object-fit: contain;
+}
+
+.goalContainer {
+  width: auto;
+
+  grid-row-start: 2;
+  grid-column-start: 3;
+
+  background: var(--green);
+  border: 1px solid var(--white);
+  border-radius: 10px;
+}
+
+.goalAmount {
+  position: relative;
+  top: 50px;
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 45px;
+  line-height: 55px;
+  /* identical to box height */
+  text-align: center;
+
+
+  color: var(--white);
+}
+
+.roleContainer {
+  box-sizing: border-box;
+  width: auto;
+  height: auto;
+  background: linear-gradient(180deg, var(--green) 23.32%, rgba(6, 214, 160, 0) 100%);
+  border: 1px solid var(--white);
+  border-radius: 10px;
+}
+
+.progress {
+  display: flex;
+}
+
+.progressContainer {
+  background: linear-gradient(180deg, rgba(248, 255, 229, 0.31) 25.13%, var(--blue) 100%);
+  border: 1px solid var(--white);
+  border-radius: 10px;
+  width: 320px;
+  height: 790px;
+}
+
+.planProgressHeading {
+  font-style: normal;
+  text-align: center;
+  margin-top: 30px;
+  font-weight: 800;
+  font-size: 25px;
+  line-height: 30px;
+
+  color: var(--white);
+
+  opacity: 0.85;
+}
+
+.planChoices {
+  margin-top: 40px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  gap: 23px;
+}
+
+.planChoices>button {
+  position: relative;
+
+  display: block;
+  background-color: var(--white);
+  border-radius: 20px;
+  border-style: none;
+  width: 277.33px;
+  height: 70px;
+  cursor: pointer;
+}
+
+.planChoices>button>span {
+  position: absolute;
+  top: 20px;
+  left: 80px;
+
+  color: var(--black);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 23px;
+}
+
+.progressBarContainer {
+  position: relative;
+  margin: auto;
+  margin-top: 30px;
+  width: 277.33px;
+  height: 157.17px;
+
+  background: #1B9AAA;
+  border: 3px solid #FFFFFF;
+  border-radius: 20px;
+}
+
+.progressStatus {
+  position: absolute;
+  top: 30px;
+  left: 30px;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 27px;
+
+  margin: 0;
+  width: 130px;
+  height: 70px;
+  color: var(--white);
+
+}
+
+.itemsCompleted {
+  position: absolute;
+  bottom: 50px;
+  left: 30px;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 8px;
+  line-height: 10px;
+  color: var(--white);
+}
+
+.progressBar {
+  position: absolute;
+  width: 80.33px !important;
```

Comment:
hm it's a general antipattern to use !important - I'm not gonna block on this but just know that almost all of the time, this can be implemented in a different way without needing !important

Reply:
I see. This only reason why i'm using !important is because i'm using a library for the progress bar